modules/ROOT/pages/questions/question-22.adoc

```diff
@@ -0,0 +1,38 @@
+22. Two elements default values were changed: 
```

Comment:
@ltrouton TINY-8259

Reply:
Let's not mention anything about future plans. For now, this is purely part of modernising TinyMCE's default behaviour.